package.json

```diff
@@ -20,12 +20,14 @@
 		"find-java-home": "^1.1.0",
 		"globby": "^11.0.0",
 		"html-escaper": "^3.0.0",
+		"is-zip": "^1.0.0",
+		"istextorbinary": "^5.12.0",
```

Comment:
I'm going to create a 3PP for these libraries shortly.

Reply:
istextorbinary has been approved: https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qB0000000VI8aIAG/view
Still waiting on is-zip.

Reply:
is-zip is approved: https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qB0000000VI8fIAG/view